processor/spanmetricsprocessor/processor_test.go

```diff
@@ -468,7 +475,7 @@ func newProcessorImp(mexp *mocks.MetricsExporter, tcon *mocks.TracesConsumer, de
 	}
 	return &processorImp{
 		logger:          logger,
-		config:          Config{AggregationTemporality: temporality},
+		config:          Config{AggregationTemporality: temporality, IngestLatencyMetricService: "trace-ingestion-synthetics"},
```

Comment:
maybe this should be a test value

Reply:
Yeah, I felt weird about hard-coding it, to be sure.